src/patternfly/components/Nav/nav.scss

```diff
@@ -354,180 +252,105 @@
   }
 }
 
+.pf-c-nav__item.pf-m-expanded .pf-c-nav__toggle > * {
+  transform: var(--pf-c-nav__item--m-expanded__toggle--i--Transform);
+}
 
-// List default
-.pf-c-nav__list {
-  // Item
-  > .pf-c-nav__item {
-    position: relative;
-
-    .pf-c-divider {
-      --pf-c-nav__c-divider--MarginLeft: var(--pf-c-nav__simple-list--nested__c-divider--MarginLeft);
-    }
+// Nav link
+.pf-c-nav__link {
+  position: relative;
+  display: flex;
+  align-items: baseline;
+  padding: var(--pf-c-nav__link--PaddingTop) var(--pf-c-nav__link--PaddingRight) var(--pf-c-nav__link--PaddingBottom) var(--pf-c-nav__link--PaddingLeft);
+  font-size: var(--pf-c-nav__link--FontSize);
+  font-weight: var(--pf-c-nav__link--FontWeight);
+  color: var(--pf-c-nav__link--Color);
+  background-color: var(--pf-c-nav__link--BackgroundColor);
+
+  &::before,
+  &::after {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
```

Comment:
This makes the text unselectable because it overlays the the entire link. Is there any reason you wouldn't use `width/height/background-color`?  That's typically how we create these treatments using pseudo elements.

Reply:
It's not because of the pseudo selectors. Disable them and try to select the text. It seems to be the caused by changing the `display` value and/or adding padding to an `<a>` element. Our current nav exhibits the same behavior.

Reply:
I find this method super efficient in that we don't have to specify multiple values to say, "Enable the lefthand border", like `top: 0; bottom: 0; left: 0; width: 4px; background-color: blue;`. Then for border right say `left: auto; right: 0; ` You can just say, "all border are 0 width and color blue, then give `--BorderLeftWidth` a width value to enable. It allows us to namespace active borders.

Reply:
Yeah, I still think that as a general principle, we should not introduce an invisible overlay, just for styling purposes, over the element that keeps you from interacting with the things in the element under it. Unless it's necessary of course - though it doesn't seem necessary here. We define most borders using pseudo elements with `width/height/background-color` in other components, so it's also a more consistent approach as well.

Reply:
We'll need to update [tabs](https://patternfly-next-pr-2757.surge.sh/documentation/core/components/tabs) as well

Reply:
And looks like [buttons](https://pf4.patternfly.org/documentation/core/components/button)

https://github.com/patternfly/patternfly/issues/2893